custom/icds_reports/reports/awc_reports.py

```diff
@@ -1010,9 +1010,31 @@ def get_awc_report_beneficiary(start, length, draw, order, awc_id, month, two_be
         open_in_month=1,
         valid_in_month=1,
         age_in_months__lte=72
-    ).order_by('-month', order)
+    )
+
+    if 'current_month_nutrition_status' in order:
+        sort_order = {
+            'Severely underweight': 1,
+            'Moderately underweight': 2,
+            'Normal weight for age': 3,
+            'Data Not Entered': 4
+        }
+        reverse = '-' in order
+        data = sorted(
+            data,
+            key=lambda val: (sort_order[
+                get_status(
+                    val.current_month_nutrition_status,
+                    'underweight',
+                    'Normal weight for age'
+                )['value']
+            ]),
+            reverse=reverse
+        )
```

Comment:
would there be any performance issues by having to load all the data into memory? how many rows do we expect here versus the `length` of the data set

Reply:
@czue This is depend how many beneficiaries have one AWC.